src/_includes/docs/debug/debug-flow-ios.md

```diff
@@ -4,7 +4,7 @@ To generate the needed iOS platform dependencies,
 run the `flutter build` command.
 
 ```console
-flutter build ios --config-only --no-codesign --debug
+$ flutter build ios --config-only --no-codesign --debug
```

Comment:
Should we include command output as well?

Reply:
@domesticmouse : It's in the next code block. I split them so readers can copy the command without the response.